package.json

```diff
@@ -11,9 +11,11 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.6.0",
+    "popup": "^0.0.3",
```

Comment:
Is there need for both libraries popup and react-popup. If needed that is okay, but if possible lower number of libraries.